### PR TITLE
6 patch api reviews

### PR DESCRIPTION
**Pull request by AdaamT**
Error handling added for patch request

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -70,7 +70,60 @@ describe("api/reviews/:review_id", () => {
     });
   });
   describe("PATCH", () => {
-    test("should add a response body to the request object ", () => {});
+    test("Using the request body, should update the review object vote property ", () => {
```
Comment (pkaodev) [inline __tests__/app.test.js]: 73 - should it also return the updated review object?

File: __tests__/app.test.js [hunk 2]
```diff
@@ -70,7 +70,60 @@ describe("api/reviews/:review_id", () => {
...
+            owner: "philippaclaire9",
+            review_img_url:
+              "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+            review_body: "Fiddly fun for all the family",
+            category: "dexterity",
+            created_at: expect.any(String),
+            votes: 6,
+          };
+          expect(updatedReview).toEqual(testReview);
+          expect(updatedReview.votes).toBe(6);
```
Comment (pkaodev) [inline __tests__/app.test.js:94]: 94 - is this needed?  Is this being checked for with line 93?

**Review by pkaodev — CHANGES_REQUESTED** [review]
Tests are looking good, you've covered all the potential errors.

Just a couple of small changes (see comments).  No need for another nchelp PR for those.

File: __tests__/app.test.js (hunk 2 above)
Comment (AdaamT) [reply __tests__/app.test.js:94]: Haz told me to have that 2nd expect statement in

**Pull request merged**